app/src/main/res/values/strings.xml

```diff
@@ -100,7 +100,7 @@
     <string name="settings_key_lock_barcode_orientation" translatable="false">pref_lock_barcode_orientation</string>
     <string name="settings_keep_screen_on">Keep screen on while viewing a card</string>
     <string name="settings_key_keep_screen_on" translatable="false">pref_keep_screen_on</string>
-    <string name="settings_disable_lockscreen_while_viewing_card">Disable lock screen while viewing a card</string>
+    <string name="settings_disable_lockscreen_while_viewing_card">No lock screen while displaying cards</string>
```

Comment:
This is in the settings screen. I really do not think changing the tense here is correct. Every single other app I use uses this tense.

![image](https://user-images.githubusercontent.com/1885159/113521543-6a8c1680-959a-11eb-9dc8-663229f53c7d.png)

Reply:
On that note I should make these things multiline, so you can actually read the whole thing...

Reply:
This issue is still there.

Reply:
I went with shorter options to prevent spanning.